tests/utils/test_ctftime.py

```diff
@@ -173,3 +184,53 @@ def test_ctf_ended():
             with ctftime.ended():
                 assert ctf_ended() is True
     destroy_ctfd(app)
+
+
+def test_unix_time():
+    """
+    Tests that the unix_time function returns the correct value and fails gracefully for strange inputs
+    """
+    assert unix_time(datetime(2017, 1, 1)) == 1483228800
+    assert type(unix_time(datetime(2017, 1, 1))) == int
+    assert unix_time(None) is None
+    assert unix_time("test") is None
+    assert unix_time(1) is None
+
+
+def test_unix_time_millis():
+    """
+    Tests that the unix_time function returns the correct value and fails gracefully for strange inputs
+    """
+    # Aware datetime object
+    assert unix_time_millis(datetime(2017, 1, 1)) == 1483228800000
+    assert type(unix_time_millis(datetime(2017, 1, 1))) == int
+    assert unix_time_millis(None) is None
+    assert unix_time_millis("test") is None
+    assert unix_time_millis(1) is None
+
+
+def test_unix_time_to_utc():
+    """
+    Tests that the unix_time function returns the correct value and fails gracefully for strange inputs
+    """
+    assert unix_time_to_utc(0) == datetime(1970, 1, 1, tzinfo=timezone.utc)
+    assert unix_time_to_utc(1483228800) == datetime(2017, 1, 1, tzinfo=timezone.utc)
```

Comment:
Note this fails ont he previous commit because there did not used to be a timezone and there is one now.

Reply:
These times are different

```
>>> dt1 = datetime.utcfromtimestamp(1483228800)
>>> dt2 = datetime(2017, 1, 1, tzinfo=timezone.utc)
>>> dt1
datetime.datetime(2017, 1, 1, 0, 0)
>>> dt1.timestamp()
1483246800.0
>>> dt2.timestamp()
1483228800.0
```